src/api/objects/ObjectAPI.js

```diff
@@ -189,6 +189,16 @@ define([
         });
     };
 
+    ObjectAPI.prototype.search = function* (options) {
```

Comment:
We need some jsdoc here to describe how this works both for callers, and for search providers.

Reply:
Added jsdoc